src/events/VoiceState.ts

```
@@ -15,40 +17,59 @@ export class Voice extends EventHandler<Events.VoiceStateUpdate> {
+                return
+            }
+            const slot = await database.findSave(member.id, 0)
+            let disableRequests: boolean
+            if (slot?.requestsEnabled !== undefined) {
+                disableRequests = !slot.requestsEnabled
+            } else disableRequests = false
             const channel = await guild.channels.create({
-                name: `${member.user.globalName} channel`,
+                // todo: per server templates for channel name
+                name: slot?.name ?? `${member.user.globalName} channel`,
+                bitrate: slot?.bitrate ?? 64000,
```

ESLint

[error] Invalid group length in numeric value.

(unicorn/numeric-separators-style)